rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/RMRest.java

```diff
@@ -191,10 +191,11 @@ public String loginWithCredential(@MultipartForm LoginForm multipart) throws RMA
 
     @Override
     public String getLoginFromSessionId(String sessionId) {
-        if (sessionId != null && sessionStore.exists(sessionId)) {
+        try {
             return sessionStore.get(sessionId).getUserName();
+        } catch (Exception e) {
+            return "";
```

Comment:
Could we log it?

Reply:
it is not meant to be logged, but an information sent to any rest client whether this session is valid or not. It's the responsibility of the client to log it. I can log at debug level eventually

Reply:
Ok